### PR TITLE
Feature request template

### PR DESCRIPTION
**Pull request by balopat**
Inspired by the SciPy FR template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_requet.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (dstrain115) [inline .github/ISSUE_TEMPLATE/feature_requet.md:1]: Name of this file is mispelled.

File: .github/ISSUE_TEMPLATE/feature_requet.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+**Describe alternatives/workarounds you've considered**
+
+
+**Additional context (e.g. screenshots)**
+
+
+**What is the urgency from your perspective for this issue? Is it blocking important work?**
+<!-- Please choose one and remove the others -->
+
+P1 - this should land no later than a week
```
Comment (dstrain115) [inline .github/ISSUE_TEMPLATE/feature_requet.md:25]: It seems like all of these priorities should be one less.  P0 = less than a week, etc.  That seems closer to how we rank things internally.

File: .github/ISSUE_TEMPLATE/feature_requet.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+---
+
+**Is your feature request related to a use case or problem? Please describe.**
+
+
+**Describe the solution you'd like**
+
+
+**Describe alternatives/workarounds you've considered**
```
Comment (dstrain115) [inline .github/ISSUE_TEMPLATE/feature_requet.md:16]: Should we mark this and the next one as optional?  I don't want to put too many barriers in the way of people asking for features.

Comment (balopat) [reply .github/ISSUE_TEMPLATE/feature_requet.md:16]: SGTM!

File: .github/ISSUE_TEMPLATE/feature_requet.md (hunk 1 above)
Comment (balopat) [reply .github/ISSUE_TEMPLATE/feature_requet.md:1]: ooops! Good eye!

File: .github/ISSUE_TEMPLATE/feature_requet.md (hunk 2 above)
Comment (balopat) [reply .github/ISSUE_TEMPLATE/feature_requet.md:25]: I'm open to it. 
My thinking was that I'd use a single priority labelling for both bugs and features - and I just wanted to hold P0 for hotfixes which I thought would only be for bugs - I can hardly imagine the lack of a new feature being critical/harmful that needs a hotfix immediately (the class I would use for P0) while bugs can reach that level.

Comment (dstrain115) [reply .github/ISSUE_TEMPLATE/feature_requet.md]: Ah, that's a good point.  In the same vein, do we really have features that need to be done within a week?!?

**Pull request merged**